<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="AuthorPic">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>471</width>
      <height>111</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/images/author.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="LogoPic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>741</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/images/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="BasicBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>102</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Basic</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="TrackerBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>102</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Tracker</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="WandererBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Wanderer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MoveBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/images/arrow.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/images/x.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="UndoBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/images/undo.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="RobotBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/images/robot.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="BombBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/images/bomb.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="WallBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/images/wall.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="background" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>571</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AnimateBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/images/animate.png</normaloff>
      <normalon>:/images/stop.png</normalon>:/images/animate.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>175</width>
      <height>100</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
